txp/Pos/Txp/Logic/Local.hs

```diff
@@ -47,12 +54,15 @@ txProcessTransaction
 txProcessTransaction itw@(txId, txAux) = do
     let UnsafeTx {..} = taTx txAux
     tipDB <- GS.getTip
+    bootEra <- gsIsBootstrapEra
+    bootHolders <- views (lensOf @GenesisUtxo) $ getKeys . utxoToStakes . unGenesisUtxo
```

Comment:
Well, you could put `genesisStakeholdersM` to `txp`. But ok, it's a small function so I don't insist.